modules/context/context.go

```diff
@@ -127,6 +137,7 @@ func RedirectToUser(ctx *Context, userName string, redirectUserID int64) {
 	if ctx.Req.URL.RawQuery != "" {
 		redirectPath += "?" + ctx.Req.URL.RawQuery
 	}
+	// FIXME: Why not Fragment
```

Comment:
Maybe we do not have fragment in URL (as above)

ps: should the `Replace` work with escaped path?

```
redirectPath := strings.Replace(
		ctx.Req.URL.Path,  // escaped or not?
		userName, // escaped or not?
		user.Name,  // escaped ?
		1,
	)
```

Reply:
```suggestion
```

Reply:
It needs to be the escaped path or it will have to be escaped later.

Reply:
I mean line 131 in this file, I can see other code was changed to use `escaped` mechanism, but this one wasn't changed.